src/storage/txn/scheduler.rs

```diff
@@ -880,9 +1031,15 @@ impl<E: Engine, L: LockManager> Scheduler<E, L> {
             .unwrap();
     }
 
+    fn is_undetermined_error(_e: &tikv_kv::Error) -> bool {
+        // TODO: If there's some cases that `engine.async_write` returns error but it's
```

Comment:
If this happened, it should be a bug? Because it will affect correctness for other requests.

Reply:
I'm afraid this problem already exists, and there's possible to be more such cases in the future.

* https://github.com/pingcap/tidb/issues/21355
* https://github.com/tikv/tikv/issues/9113
* https://github.com/tikv/tikv/pull/12369#discussion_r852017654

Reply:
I just left an empty function here since I think we may need to handle these possibilities in the future.